components/__tests__/SafeHtml.test.tsx

```
@@ -0,0 +1,83 @@
+
+  it('handles empty HTML string', () => {
+    const { container } = render(<SafeHtml html="" />)
+    expect(container.firstChild).toBeInTheDocument()
+    expect(container.innerHTML).toBe('<div></div>')
+  })
+
+  it('works in server-side environment', () => {
+    // Temporarily mock window as undefined to simulate server environment
+    const originalWindow = global.window
+    // @ts-ignore
+    delete global.window
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)